tdrs-backend/tdpservice/settings/common.py

```diff
@@ -71,6 +72,7 @@ class Common(Configuration):
         "tdpservice.middleware.NoCacheMiddleware",
     )
 
+    APP_NAME = "dev"
```

Comment:
This is a default for local environments?

Reply:
Yes, so I could test with local stack that prefixes were working.